apps/web/src/components/data-table/data-table.tsx

```diff
@@ -110,7 +111,7 @@ export function DataTable<TData, TValue>({
           </TableBody>
         </Table>
       </div>
-      <DataTablePagination table={table} />
```

Comment:
Keep the `DataTablePagination` here, you can get the `limit` via the searchParams instead of passing it as `children`.

Reply:
@mxkaske I get that, but the reason why I'm using it as a child is to prevent prop-drilling the number of data results.
Remember we need the number of data results to know if there's a next page or not.

Reply:
Yep, gotcha. Let’s keep that for now and iterate later.